scheduler/pkg/kafka/gateway/manager.go

```diff
@@ -86,9 +90,10 @@ func (cm *ConsumerManager) createKafkaConfigs(kafkaConfig *ManagerConfig) error
 		return err
 	}
 
-	producerConfigAsJSON, err := json.Marshal(&producerConfigMap)
+	producerConfigWithoutSecrets := config.WithoutSecrets(producerConfigMap)
+	producerConfigAsJSON, err := json.Marshal(&producerConfigWithoutSecrets)
```

Comment:
`producerConfigAsJSON` -> `producerConfigWithoutSecretsAsJSON`?

Reply:
Shorter names are easier to read, particularly with small scopes of usage

Reply:
- `producerConfigMap`
- `producerConfigMapSafe`
- `producerConfigMapSafeJSON`
?

Reply:
Fixed in https://github.com/SeldonIO/seldon-core/pull/4966/commits/1dd883adcfbcf70874337f7729c85ea42b975a0d